src/app/campaigns/components/farm-welcome/farm-welcome.component.html

```diff
@@ -242,15 +236,15 @@
 <div class="welcome_footer_lunch text-center">
   <div class="col-11 col-md-7" style="margin: auto">
     <div class="pt-5 mb-2 welcome_footer_title" style="margin: auto">
-      Launch your Adpool Now !
+     {{ 'Launch_your_Adpool_Now' | translate }}
     </div>
     <div class="row mb-4 welcome_description_long">
-      Adpool is an innovative tool for advertisers looking to maximize the
-      impact of their marketing campaigns. It allows advertisers to solicit
-      influencers to speak about their brand or product, while also compensating
-      them with the SaTT token. By applying to create an Adpool, our experienced
-      team will contact you to guide and assist you in setting up your campaign.
-      Trust Adpool for a successful marketing campaign !
+      {{ 'Launch_your_Adpool_Now_l1' | translate }}
+      {{ 'Launch_your_Adpool_Now_l2' | translate }}
+      {{ 'Launch_your_Adpool_Now_l3' | translate }}
+      {{ 'Launch_your_Adpool_Now_l4' | translate }}
+      {{ 'Launch_your_Adpool_Now_l5' | translate }}
+      {{ 'Launch_your_Adpool_Now_l6' | translate }}
```

Comment:
In order to facilitate localization and internationalization, could we please place this entire block of text into a single i18n translation variable? This will allow us to translate the entire block of text at once, rather than dealing with individual sentences, ensuring a more consistent translation. Thank you.